data/uhc_pack/functions/lobby/mark_players.mcfunction

```diff
@@ -0,0 +1,8 @@
+# Add the 'playing' tag for all players who are participating in the UHC
+# This is important for the death function
+tag @a[team=!spectate,tag=!playing] add playing
+tag @a[team=spectate,tag=playing] remove playing
+
+# Count the amount of people that currently have the "playing" tag.
+execute store result score Players uhcState run execute if entity @a[tag=playing]
```

Comment:
I don't completely understand why this is temporarily stored against `Players uhcState` before then transferring it to another scoreboard.